elements/pf-chip/test/pf-chip.spec.ts

```diff
@@ -45,19 +45,21 @@ describe('<pf-chip>', async function() {
       await expect(element).to.be.accessible();
     });
 
-    describe('calling focus', function() {
+    describe('calling focus() on the element', function() {
       beforeEach(() => element.focus());
-      it('should focus', function() {
-        expect(activeElement(element)?.getAttribute('aria-label'))
-          .to.equal(element.accessibleCloseLabel);
+      it('should focus', async function() {
+        const snapshot = await a11ySnapshot();
+        expect(snapshot.children?.at(0)?.name).to.equal(element.accessibleCloseLabel);
+        expect(snapshot.children?.at(0)?.focused).to.be.true;
       });
     });
 
     describe('pressing Tab', function() {
       beforeEach(tab);
-      it('should focus', function() {
-        expect(activeElement(element)?.getAttribute('aria-label'))
-          .to.equal(element.accessibleCloseLabel);
+      it('should focus', async function() {
+        const snapshot = await a11ySnapshot();
+        expect(snapshot.children?.at(0)?.name).to.equal(element.accessibleCloseLabel);
+        expect(snapshot.children?.at(0)?.focused).to.be.true;
       });
 
```

Comment:
We need a test for accessible description as well.

Reply:
I'm not sure we can test for this, the snapshot doesn't surface the aria description on the button's container
here's how it looks in ff's ax tree inspector:
![image](https://github.com/patternfly/patternfly-elements/assets/1466420/63f0863e-cfdf-4a87-a639-ce4480734ed6)

Reply:
I added @hellogreg as a reviewer as well to see if this resolves the Win/FF and Win/Edge testing issues he's had with chip.

Reply:
ok, if not, since it appears we won't be able to apply the description across roots, that means we can't use pf-button until browser vendors ship a solution for x-root aria. This creates two maintenance problems:
1. duplicating pf-button here
2. what to do when browser support improves

for (1), i'd like to suggest we give some thought to creating some pf-button abstractions that can sit in a shadow template, maybe an element-position directive that applies styles based on attributes? barring that, to see whether or not we could just apply the shadow stylesheet from pf-button here
for (2), I think we should leave a comment in the template with a note to check back in q4, depending on how the browser vendors get on

WDYT?

Reply:
So, the good news is that chips in this deploy preview are navigable via keyboard in all browsers.

The not-so-good news is what's announced when screen readers are enabled.

Typically (e.g., in the live version of chip), the chip's contents get announced:

<img width="698" alt="Screencap of live chip with keyboard focus" src="https://github.com/patternfly/patternfly-elements/assets/642695/7211a57c-e741-4d98-a169-02158597ba5f">

However, in the current version of this deploy preview, screen readers just say "close, button":

<img width="693" alt="Screencap of deploy preview chip with focus" src="https://github.com/patternfly/patternfly-elements/assets/642695/54bfb93e-5dd6-4938-8735-586e439f04f8">

Reply:
@hellogreg what's the situation with a single chip? I'm wondering if the issue is the `aria-describedby` not being identified.

Reply:
Yep, you're right, @nikkimk . Looks like the description isn't being identified for any chips.

Reply:
this should be ready for another review